packages/test/test-end-to-end-tests/src/test/agentScheduler.spec.ts

```diff
@@ -4,69 +4,69 @@
  */
 
 import { strict as assert } from "assert";
-import { IAgentScheduler } from "@fluidframework/agent-scheduler";
-import { IContainer } from "@fluidframework/container-definitions";
-import { agentSchedulerId } from "@fluidframework/container-runtime";
+import { AgentSchedulerFactory, IAgentScheduler } from "@fluidframework/agent-scheduler";
+import { IContainer, IProvideRuntimeFactory } from "@fluidframework/container-definitions";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
-import { ITestObjectProvider, timeoutPromise, defaultTimeoutDurationMs } from "@fluidframework/test-utils";
-import { describeFullCompat, ITestDataObject } from "@fluidframework/test-version-utils";
+import {
+    ITestObjectProvider,
+    TestContainerRuntimeFactory,
+} from "@fluidframework/test-utils";
+import { describeFullCompat } from "@fluidframework/test-version-utils";
+
+const runtimeFactory: IProvideRuntimeFactory = {
+    IRuntimeFactory: new TestContainerRuntimeFactory(AgentSchedulerFactory.type, new AgentSchedulerFactory()),
+};
 
 describeFullCompat("AgentScheduler", (getTestObjectProvider) => {
-    let leaderTimeout = defaultTimeoutDurationMs;
     let provider: ITestObjectProvider;
+
+    const createContainer = async (): Promise<IContainer> =>
+        provider.createContainer(runtimeFactory);
+
+    const loadContainer = async (): Promise<IContainer> =>
+        provider.loadContainer(runtimeFactory);
+
     beforeEach(() => {
         provider = getTestObjectProvider();
-        leaderTimeout = provider.driver.type === "odsp" ? 5000 : defaultTimeoutDurationMs;
     });
-    const leader = "leader";
+
     describe("Single client", () => {
         let scheduler: IAgentScheduler;
 
         beforeEach(async () => {
-            const container = await provider.makeTestContainer({
-                runtimeOptions: { addGlobalAgentSchedulerAndLeaderElection: true },
-            });
-            scheduler = await requestFluidObject<IAgentScheduler>(container, agentSchedulerId);
```

Comment:
can you add/keep/copy a couple tests that still set addGlobalAgentSchedulerAndLeaderElection and validate the leader taks?

Reply:
Chatted with Tony -- leader.spec.ts still does this.  I'll make sure we keep at least one test with the flag until we're ready to remove.